bimsb/packages/staging.scm

```diff
@@ -3024,3 +3024,25 @@ the xlxs package.  This release corresponds to POI 3.10.1.")
        "MuSiC is a deconvolution method that utilizes cross-subject scRNA-seq
 to estimate cell type proportions in bulk RNA-seq data.")
       (license license:gpl3))))
+
+(define-public r-dsa
+  (let ((commit "f181f549fd86feb58955f11e53ee38da67ac4f5e"))
```

Comment:
I'd suggest adding a "revision" variable as well, so that later we can use `git-version`.